tests/JMS/Serializer/Tests/Metadata/Driver/yml/Discriminator.ObjectWithXmlAttributeDiscriminatorParent.yml

```diff
@@ -0,0 +1,8 @@
+JMS\Serializer\Tests\Fixtures\Discriminator\ObjectWithXmlAttributeDiscriminatorParent:
+    discriminator:
+        field_name: type
+        map:
+            child: JMS\Serializer\Tests\Fixtures\Discriminator\ObjectWithXmlAttributeDiscriminatorChild
+    xml_discriminator:
+        attribute: true
+        cdata: false
```

Comment:
Having the YAML format being wildly different to annotations and xml feels odd to me, but if you insist, i will change it...